src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Triggers/MessageToStringConverter.cs

```diff
@@ -1,72 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Globalization;
-using System.IO;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.Azure.ServiceBus;
-using Microsoft.Azure.ServiceBus.InteropExtensions;
-
-namespace Microsoft.Azure.WebJobs.ServiceBus.Triggers
-{
-    internal class MessageToStringConverter : IAsyncConverter<Message, string>
-    {
-        public async Task<string> ConvertAsync(Message input, CancellationToken cancellationToken)
-        {
-            if (input == null)
-            {
-                throw new ArgumentNullException("input");
-            }
-            if (input.Body == null)
-            {
-                return null;
-            }
-            Stream stream = new MemoryStream(input.Body);
-
-            TextReader reader = new StreamReader(stream, StrictEncodings.Utf8);
-            try
-            {
-                cancellationToken.ThrowIfCancellationRequested();
-                try
-                {
-                    return await reader.ReadToEndAsync();
-                }
-                catch (DecoderFallbackException)
-                {
-                    // we'll try again below
-                }
-
-                // We may get here if the message is a string yet was DataContract-serialized when created. We'll
```

Comment:
The new Message->String converter you added is just a one liner. How does it differ from what we were doing before - are you sure there are no breaking changes here?

Reply:
It shouldn't break, I added tests for different conversions.